setup.py

```diff
@@ -88,14 +88,50 @@ def compilation_help():
     sys.argv.remove ("-warnings")
 
 if 'cython' in sys.argv:
+    # compile .pyx files
     # So you can `setup.py cython` or `setup.py cython install`
     try:
-        from Cython.Build import cythonize
+        from Cython.Build.Dependencies import cythonize_one
     except ImportError:
         print("You need cython. https://cython.org/, pip install cython --user")
+        sys.exit(1)
+
+    import glob
+    pyx_files = glob.glob(os.path.join('src_c', 'cython', 'pygame', '*.pyx')) + \
```

Comment:
Does this handle `.pxd` files changing as well?

Reply:
Fixed. It would be useful if this also worked for the regular header files.